File: Languages/npeg_php/TokenMatch.php
<?
namespace RobustHaven\Text\Npeg;

class TokenMatch 
{
	private $name;
	private $start;
	private $end;
	
	function __construct($name,$start,$end) 
	{
       $this->name = $name;
       $this->start = $start;
       $this->end = $end;
    }
   
	public function setName( $value )
	{
		$this->name = $value;
	}

	public function getName()
	{
		return $this->name;
	}

	public function setStart( $value )
	{
		$this->start = $value;
	}

	public function getStart()
	{
		return $this->start;
	}
	
	public function setEnd( $value )
	{
		$this->end = $value;
	}

	public function getEnd()
	{
		return $this->end;
	}
}

?>

File: Languages/npeg_java/robusthaven/robusthaven/text/TokenMatch.java
package robusthaven.text;

public class TokenMatch {
    String m_name;
    long m_start, m_end;
    
    public final String getName() {
	return m_name;
    }

    public long getStart() {
	return m_start;
    }

    public long getEnd() {
	return m_end;
    }

    public TokenMatch(String name, long start, long end) {
	m_name = name;
	m_start = start;
	m_end = end;
    }
}